observability/logging/log_storage/about-log-storage.adoc

```diff
@@ -12,9 +12,7 @@ You can use an internal Loki or Elasticsearch log store on your cluster for stor
 [id="log-storage-overview-types"]
 == Log storage types
 
-Loki is a horizontally scalable, highly available, multi-tenant log aggregation system offered as an alternative to Elasticsearch as a log store for the {logging}.
-
-Elasticsearch indexes incoming log records completely during ingestion. Loki only indexes a few fixed labels during ingestion and defers more complex parsing until after the logs have been stored. This means Loki can collect logs more quickly.
+Loki is a horizontally scalable, highly available, multi-tenant log aggregation system offered as a log store for the {logging}. OpenShift {logging-uc} provides Loki as a GA log store that can be visualized with the OpenShift {ObservabilityShortName} UI. The Loki configuration provided by OpenShift {logging-uc} is a short term log store designed to enable users to perform fast troubleshooting with the collected logs. For that purpose, the OpenShift {logging-uc} configuration of Loki has short term storage, and is optimized for very recent queries. For long term storage or queries over a long time period, users should look to log stores external to their OpenShift cluster.
```

Comment:
This needs to either be conditionalized for use in 6.0, or perhaps simply made into a snippet.

Reply:
@libander Thanks for the edit and agreed. That's why I have raised it against the 6.0 branch.

Reply:
@libander I have created 2 snips 1 for post 6x and another for pre 6x. Placing the pre 6x one in its place. Once the docs are structured for 6x, we can user the post 6x snip for Loki introduction.